Documentation/CRDs/Block-Storage/ceph-block-pool-crd.md

```diff
@@ -232,6 +232,12 @@ stretched) then you will have 2 replicas per datacenter where each replica ends
     !!! note
         A value of 0 disables the quota.
 
+* `placementGroupCount`: Set a custom PG count for the pool. By default, the PGs will be managed by the Ceph PG autoscaler. See the [Ceph documentation](https://docs.ceph.com/en/latest/rados/operations/pools/#create-a-pool) for more info.
```

Comment:
Do we need this setting? The user can currently set the pg count with this, right? Any pool setting found [here](https://docs.ceph.com/en/latest/rados/operations/pools/#set-pool-values) can be set with the `parameters`.
```yaml
apiVersion: ceph.rook.io/v1
kind: CephBlockPool
metadata:
  name: replicapool
  namespace: rook-ceph # namespace:cluster
spec:
  parameters:
    pg_num: "32"
    nopgchange: "1"
```
I'm not sure if the `nopgchange` is the same as disabling the autoscaler, so perhaps the `autoscaleMode` would still be required in the schema. 

Where we can find a way to use the `parameters`, it's nice that we wouldn't need to update the schema.

Reply:
In summary, nopgchange and autoscalemode are both settings that can be used in Ceph to control the scaling behavior of the storage cluster, but they operate at different levels and have different effects. nopgchange is used to control whether individual PGs can be moved between OSDs, while autoscalemode is used to control how the cluster automatically scales its resources based on the workload.

Reply:
maybe we can add "pg_nm" and "autoscalemode" to the **parameters**, no need to add to the schema?

Reply:
I see documentation for `pg_autoscale_mode` here: https://docs.ceph.com/en/latest/rados/operations/placement-groups/#autoscaling-placement-groups

Reply:
@zhucan please stop ongoing conversations as resolved on GitHub. It takes ages to find where past conversations have been started. Instead, please respond to the question.